### PR TITLE
Create app/version with package

### PR DESCRIPTION
**Pull request by zheng1**
1. Improved version information
2. Add default helm/vmbased repo
3. Refactor `openpitrix.io/openpitrix/pkg/devkit/app` to `openpitrix.io/openpitrix/pkg/devkit/opapp`
4. Add repo ReaderWriter
5. Support CreateApp/CreateAppVersion with a package
6. Add byte flag support to opctl
7. Add `default_draft_status`  to global_config
8. Auto-sync app_vesion package information into app_version table
9. Support create repo even without index.yaml on it

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -23,6 +24,18 @@ services:
     ports:
      - "12379:2379" # for unit-test & debug
     container_name: "openpitrix-etcd"
+  openpitrix-minio:
```
Comment (chilianyi) [inline docker-compose.yml]: Yaml is needed when deploy in kubernetes.

File: pkg/repoiface/reader.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	"bytes"
+	"context"
+
+	"github.com/pkg/errors"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/provenance"
+	"k8s.io/helm/pkg/repo"
+
+	"openpitrix.io/openpitrix/pkg/repoiface/wrapper"
+
```
Comment (chilianyi) [inline pkg/repoiface/reader.go]: fmt

Comment (zheng1) [reply pkg/repoiface/reader.go]: done

**Comment by zheng1** [comment]
Fixed #574 #571

**Pull request merged**